src/main/java/cam72cam/immersiverailroading/ImmersiveRailroading.java

```diff
@@ -110,6 +111,8 @@ public void commonEvent(ModEvent event) {
 				MultiblockRegistry.register(CastingMultiblock.NAME, new CastingMultiblock());
 				IRFuzzy.applyFallbacks();
 				break;
+			case START:
+				PhysicsThread.start();
```

Comment:
This logic should be moved into PhysicsThread.run.  It can check if the executor is started in there.